payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/resources/LpmRepository.kt

```diff
@@ -53,6 +53,7 @@ class LpmRepository constructor(
 ) {
     private val lpmSerializer = LpmSerializer()
     private val serverInitializedLatch = CountDownLatch(1)
+    var serverSpecLoadingState: ServerSpecState = ServerSpecState.Uninitialized
```

Comment:
Does this not need to be a flowable?

Reply:
no this one doesn't because nothing observes it.